src/wallet/output_basic.h

```diff
@@ -0,0 +1,63 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef __WALLET_OUTPUT_BASIC_H__
+#define __WALLET_OUTPUT_BASIC_H__
+
+#include "core/models/payloads/transaction.h"
+#include "core/models/signing.h"
+#include "wallet/wallet.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Send basic transaction which transfers IOTA tokens to an address
+ *
+ * @param[in] w A wallet instance
+ * @param[in] sender_addr The sender address
+ * @param[in] sender_keypair The sender private key
+ * @param[in] send_amount The amount to transfer
+ * @param[in] recv_addr The receiver address
+ * @param[out] msg_res The response of the transfer
+ *
+ * @return int 0 on success
+ */
+int wallet_basic_send(iota_wallet_t* w, address_t* sender_addr, ed25519_keypair_t* sender_keypair,
+                      uint64_t const send_amount, address_t* recv_addr, res_send_message_t* msg_res);
```

Comment:
the same as `wallet_alias_create_send`, please revert this change.
fo consistancey, rename to `wallet_basic_output_send`?
```suggestion
int wallet_basic_send(iota_wallet_t* w, bool sender_change, uint32_t sender_index,
                      uint64_t const send_amount, address_t* recv_addr, res_send_message_t* msg_res);
```